env/blog.env

```diff
@@ -0,0 +1,4 @@
+LD_PROJECT_TYPE="blog"
+FRONTEND_REPOSITORY=${FRONTEND_REPOSITORY:-https://github.com/drunomics/lupus-decoupled-nuxt3-demo-shadcn}
+ADDITIONAL_COMPOSER_PACKAGES=${ADDITIONAL_COMPOSER_PACKAGES:-drupal/lupus_decoupled_blog}
```

Comment:
why do we need this translation step. we can simply set ADDITIONAL_RECIPES=lupus_decoupled_blog via env-var, composer-require it and install it? Let's keep things simple.

Reply:
I don't see much difference between ADDITIONAL_COMPOSER_PACKAGES and ADDITIONAL_RECIPES. Either way we need to run composer to install those recipes first.

Reply:
I reworked the process, so all recipes - including the base `lupus_decoupled_recipe` - must be added to `$RECIPES` and I got rid of `$ADDITIONAL_RECIPES`. The reason is that way we can control the order of recipes. Before the `lupus_decoupled_recipe` was always applied AFTER `$ADDITIONAL_RECIPES` . Which doesn't make sense if we add e.g. `lupus_decoupled_blog` to `$ADDITIONAL_RECIPES` , it would apply the base recipe twice then (should not have an effect currently, but is stupid)

Reply:
It seems that one var for recipes is enough